soundpool/example/linux/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
No support for Linux (and Windows) so files for these platforms should be removed.